airbyte-integrations/connectors/source-ebay-finance/manifest.yaml

```diff
@@ -0,0 +1,471 @@
+version: 6.41.5
+
+type: DeclarativeSource
+
+description: |-
+  Website: https://www.ebay.com/
+  Documentation: https://developer.ebay.com/api-docs/sell/finances/overview.html
+
+check:
+  type: CheckStream
+  stream_names:
+    - transactions
+
+definitions:
+  streams:
+    transactions:
+      type: DeclarativeStream
+      name: transactions
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /sell/finances/v1/transaction
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - transactions
+          schema_normalization: Default
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            field_name: offset
+            inject_into: request_parameter
+          page_size_option:
+            type: RequestOption
+            field_name: limit
+            inject_into: request_parameter
+          pagination_strategy:
+            type: OffsetIncrement
+            page_size: 1000
+            inject_on_first_request: true
```

Comment:
PR Fix as suggested in [review here](https://github.com/airbytehq/airbyte/pull/56920#pullrequestreview-2730204430): 

1000 is the max pagination limit ebay finances API allows